internal/ir/xds_test.go

```diff
@@ -1421,6 +1439,7 @@ func TestValidateHealthCheck(t *testing.T) {
 					},
 				},
 				&OutlierDetection{},
+				ptr.To[uint32](10),
```

Comment:
@arkodg not sure why it was needed to set default value in panic thresholds here and in all test below (since it is supposed to be an optional field). But otherwise it did not compile